src/main.rs

```diff
@@ -29,15 +32,37 @@ fn handle_client(mut stream: TcpStream) {
                     continue 'read;
                 }
             };
+
+            let mut pos = 0;
+            let mut buffer = match File::create("face.jpg") {
+                Ok(file) => file,
+                Err(error) => {
+                    error!("file create failed: {}", error);
+                    continue 'read;
+                },
+            };
+            while pos < request.image.len() {
+                let bytes_written = match buffer.write(&request.image[pos..]) {
+                    Ok(size) => size,
+                    Err(error) => {
+                        error!("write failed: {}", error);
+                        continue 'read;
+                    },
+                };
+                pos += bytes_written;
+            }
+
             let response = Response::Accept{
-                text: request.text,
+                expression: Expression::Anger,
```

Comment:
The response shouldn't be an expression, it should be a boolean (is this face angry? -> yes/no)